.circleci/config.yml

```diff
@@ -82,6 +82,22 @@ commands:
             S3_DIR: << parameters.s3_dir >>
             S3_BUCKET: << parameters.s3_bucket >>
 
+  save_codecov:
+    description: "Persist code coverage results"
+    steps:
+      # Make a coverage file if one wasn't generated
+      # persist_to_workspace will fail if no files are matched.
+      # Use `date` to come up with a unique name for the file
+      # to avoid collisions when workspaces are merged.
+      - run: touch .coverage
+      - run: mv .coverage ".coverage.report.$(date +%s%6N)"
```

Comment:
do we need to tell `coverage.py` and `codecov` to only look for this specific file/date?

Reply:
or no... this is the per-job report right?

Reply:
right, they get merged downstream in the coverage job into a single .coverage file which codecov should look for.